tests/fixtures/util_hmailitem.js

```diff
@@ -91,10 +91,9 @@ exports.createHMailItem = (outbound_context, options, callback) => {
             callback('No hmail producted');
             return;
         }
-        for (let j=0; j<hmails.length; j++) {
-            const hmail = hmails[j];
-            hmail.hostlist = [ delivery_domain ];
-            callback(null, hmail);
+        for (const element of hmails) {
+            element.hostlist = [ delivery_domain ];
+            callback(null, element);
```

Comment:
why rename the iterable to element instead of using hmail, as was already defined?  More changed lines of code for what benefit?